lib/aruba/cucumber/command.rb

```diff
@@ -28,6 +28,21 @@
   run_simple(cmd, :fail_on_error => true, :exit_timeout => secs && secs.to_i)
 end
 
+When(/I run the following script:$/) do |file_content|
+  prepend_environment_variable('PATH', expand_path('bin') + ':')
```

Comment:
Is `#prepend_environment_variable` really needed here? It's added by default here: https://github.com/cucumber/aruba/blob/master/lib/aruba/cucumber/hooks.rb#L17.